src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day--sun calendar--month-length--28">
```

Comment:
The modifier class should use a single underscore instead of double underscore, e.g. 'calendar--start-day_sun' and 'calendar--month-length_28'